spark/src/main/scala/org/apache/comet/serde/QueryPlanSerde.scala

```diff
@@ -275,12 +284,20 @@ object QueryPlanSerde extends Logging with ShimQueryPlanSerde with CometExprShim
               .setCurrentRow(OperatorOuterClass.CurrentRow.newBuilder().build())
               .build()
           case e =>
+            val offset = e.eval() match {
+              case i: Integer => i.toLong
+              case l: Long => l
+              case _ =>
+                throw new IllegalArgumentException(
+                  "Unsupported data type for window function row/range offset")
```

Comment:
You will fail the query directly instead of falling back to Spark.

Reply:
changed to `return None` to fall back to Spark